Disable colors when not in interactive terminal

desolatorxxl opened the pull request:
Forcing colors in logrus causes color escapes sequences when logging to file or pipe.

Fixes #602.

0xERR0R commented:
Thanks for your work. I think, the idea is clear, it should print colors if it is possible and use fallback if not, right?

If I run `docker run spx01/blocky` without you change, I get color output:
![image](https://user-images.githubusercontent.com/25868513/180621139-d0c82664-f8b6-4b02-9ed1-9381a9cf9eb2.png)

with your change:
![image](https://user-images.githubusercontent.com/25868513/180621172-61d713f9-c4a4-4c96-b72c-6031ae13b9bd.png)

It works with your change, if I add "--tty" parameter (`docker run --rm --tty spx01/blocky`).

Is it the desired behavior?

desolatorxxl commented:
> Is it the desired behavior?

Yes. This way no colors will be logged by detached containers (-d) either.
One can still force colors with -t / --tty if desired.

desolatorxxl commented:
Apparently I made a mistake and changed `ForceFormatting` instead of `ForceColors`,  which both changed color and format.
I fixed that now.
It might be a good idea to not force the format as well, but that's not the scope of this PR ;)
